.github/workflows/validate-byos-multivm.yaml

```diff
@@ -152,8 +163,47 @@ jobs:
                 git status
                 git commit -m "test branch for pipeline"
                 git push https://$gitToken@github.com/${GITHUB_REPOSITORY}.git -f
+            - uses: azure/login@v1
+              id: azure-login
+              with:
+                creds: ${{ env.azureCredentials }}
+            - name: Accept Image Terms
+              id: accept-terms
+              run: |
+                  echo "accept terms for VM image"
+                  az vm image terms accept --urn redhat:rhel-byos:rhel-lvm84:latest
+            - name: Deploy an instance of Azure Database for PostgreSQL
```

Comment:
Is it better if we re-use this DB for both Standalone and Domain mode pipelines? See [existing pipeline](https://github.com/oracle/weblogic-azure/blob/main/.github/workflows/testWlsVmCluster.yml#L205)

Reply:
Good point. Fixed in [14b10ef](https://github.com/Azure/rhel-jboss-templates/pull/171/commits/14b10ef623a69488f532f1c0704766228b689fe9).